libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -182,3 +182,15 @@ export interface BaUxdEdge {
   uxd_node: number; // uxd_node id
   selected?: boolean;
 }
+
+export interface BaUxdEdge {
```

Comment:
How is the BaUxEdge connected with the scrollspy?

Reply:
Good question :D
That's probably a side effect of reading the master into this branch.